### DISCUSSION TITLE
Confusion about Auth pricing & limits

### DISCUSSION THREAD

**Discussion by Kleysley** [supabase/supabase#1729] [Questions] [answered]
Let's assume I start with the "Free" Supabase plan.
The website lists a maximum of 10,000 Auth users. But does that mean i can 

A) have a maximum of 10,000 (authenticated) users
B) have a maximum of 10,000 new users every month
C) have a maximum of 10,000 new users every month per provider
D) as many OAuth authentications as I want but only 10,000 with email & password (per month?)

A small explanation would be great, because I am a bit confused about the limits and about why there is a tick (✓) at "Social 
OAuth Providers" and if that only means there is the possibilty. The question seems dumb but it is meant seriously. 
Thank you!

**Comment by inian** [comment, marked as answer]
Hi @Kleysley, all limits are across the entire lifetime of your project - Option A you mentioned. The authentication mechanism you use to validate your users doesn't matter - you get 10k users in total.
